Closes #205 | TALPCo

ryanignatius opened the pull request:
Please name your PR after the issue it closes. You can use the following line: "Closes #ISSUE-NUMBER" where you replace the ISSUE-NUMBER with the one corresponding to your dataset.

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `nusantara/nusa_datasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_NUSANTARA_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `NusantaraConfig` for the source schema and one for a nusantara schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_nusantara --path=nusantara/nusa_datasets/my_dataset/my_dataset.py`.
- [x] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

bryanwilie reviewed (approved):
Tested and works smoothly.
Looks good to me.

Thanks for contributing @ryanignatius !

holylovenia reviewed (changes requested):
Thank you for the dataloader, @ryanignatius!

I encountered an error when I tried to use the dataloader with configuration names using the format `talpco_{src_lang}_{tgt_lang}_{schema}`. Could you please modify line 133 to `_, lang_source, lang_target = self.config.name.replace(f"_{self.config.schema}", "").split("_")` to get rid of the error?

ryanignatius commented:
> Thank you for the dataloader, @ryanignatius!
> 
> I encountered an error when I tried to use the dataloader with configuration names using the format `talpco_{src_lang}_{tgt_lang}_{schema}`. Could you please modify line 133 to `_, lang_source, lang_target = self.config.name.replace(f"_{self.config.schema}", "").split("_")` to get rid of the error?

Thanks for the feedback!
I have updated the code to fix the error as suggested.

holylovenia reviewed (approved):
Thanks for the fix, @ryanignatius! Merging now.
